app/controllers/patients_controller.rb

```diff
@@ -23,13 +19,22 @@ def edit
 
   def create
     @patient = Patient.new(patient_params)
-    @patient.save
-    respond_with(@patient)
+    if @patient.save
+      flash[:success] = "You have successfully created a patient."
+      render :edit
+    else
+      flash[:error] = "Something went creating this patient. Please try again."
```

Comment:
changed to: "Something went wrong when creating this patient. Please try again."